db/seeds.rb

```diff
@@ -5,3 +5,35 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Toilet.destroy_all
+
+Toilet.create({
+  street: "100 Maryland Ave SW",
+  city: "Washington",
+  state: "DC",
+  zip: "20224",
+  cleanliness: 4,
```

Comment:
I think everything here looks good. The only thing I would change is the cleanliness level 4. The design I'm creating only has three cleanliness levels.